static/docs/philosophy/related-technologies.md

```diff
@@ -111,10 +112,9 @@ process.
      `git clone` command. It gives more granularity on managing data and code
      separately. Hooks could be configured to make workflow simpler.
 
-   - DVC creates hardlinks (or even reflinks if they are supported). The
-     `dvc checkout` command does not actually copy data files from cache to the
-     workspace, as copying files is a heavy operation for large files (30
-     GB+).
+   - DVC creates reflinks. The `dvc checkout` command does not actually copy
```

Comment:
reflinks are rare. This phrase could be misleading. Btw, in places like this I would use a term copy-on-write links (aka reflinks)  and would put a link to some docs about them. It's not a very well known stuff.

Reply:
Added as a foot note in this document but what would be a good source to link to about it? I haven't found anything that looks authoritative enough. Wikipedia? https://en.wikipedia.org/wiki/Copy-on-write

Reply:
Link added.